src/main/java/com/google/sps/servlets/ProfileImage.java

```diff
@@ -37,9 +37,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
       String userId = userService.getCurrentUser().getUserId();
-      Key userKey = KeyFactory.createKey("User", userId);
-      Entity userEntity = datastore.get(userKey);
-      String fileName = (String) userEntity.getProperty("avatarName");
+      String fileName = userId + ".png";
```

Comment:
I would prefer a solution where you store a property on the user profile when they upload a profile picture (e.g. `hasProfilePicture`).